### PR TITLE
Fixes for test syntax and copy

### PR DESCRIPTION
**Pull request by ericaporter**
### Context

- Ticket:  n/a

### Changes proposed in this pull request
Some fixes:
1. Error message was repeating "Declaration date"
2. Delivery partner in factory was being randomised and causing Percy to create new pages
3. Syntax in spec file was incorrect

### CODE REVIEW

File: spec/features/finance/payment_breakdown_spec.rb [hunk 1]
```diff
@@ -351,21 +351,19 @@ def then_i_should_see_the_correct_payment_summary
   end
 
   def then_i_should_see_the_correct_output_fees
-    all(".finance-panel")[0] do
-      expect(page).to have_content("Output payments")
-      expect(page).to have_content(number_to_pounds(@jan_starts[:output_payments][0][:per_participant]))
-      expect(page).to have_content(@jan_starts[:output_payments][0][:participants])
-      expect(page).to have_content(number_to_pounds(@jan_starts[:output_payments][0][:subtotal]))
-    end
+    all(".finance-panel")[0]
```
Comment (asmega) [inline spec/features/finance/payment_breakdown_spec.rb:354]: not sure what this line does and can be deleted? or is there an assertion that is exists?

File: spec/features/finance/payment_breakdown_spec.rb [hunk 2]
```diff
@@ -351,21 +351,19 @@ def then_i_should_see_the_correct_payment_summary
...
   end
 
   def then_i_should_see_the_correct_uplift_fee
-    all(".finance-panel")[1] do
-      expect(page).to have_content("Uplift fee")
-      expect(page).to have_content(number_to_pounds(@jan_starts[:other_fees][:uplift][:per_participant]))
-      expect(page).to have_content(@jan_starts[:other_fees][:uplift][:participants])
-      expect(page).to have_content(number_to_pounds(@jan_starts[:other_fees][:uplift][:subtotal]))
-    end
+    all(".finance-panel")[1]
```
Comment (asmega) [inline spec/features/finance/payment_breakdown_spec.rb:362]: not sure what this line does and can be deleted? or is there an assertion that is exists?

**Review by asmega — APPROVED** [review]

File: spec/features/finance/payment_breakdown_spec.rb (hunk 1 above)
Comment (ericaporter) [reply spec/features/finance/payment_breakdown_spec.rb:354]: @asmega I thought you added it in? It was showing up in my linter as incorrect with the `do` block

Comment (ericaporter) [reply spec/features/finance/payment_breakdown_spec.rb]: Now removed, didn't need them!

**Pull request merged**